peer-dependency of ember-cli 3.2

Did you all mean to add a peer dependency of `"ember-cli": "~3.2.0"`? https://github.com/quaertym/ember-cli-dependency-checker/compare/v3.2.0...v3.3.0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R57

That seems overly specific and is quite old (July of 2018 https://github.com/ember-cli/ember-cli/releases/tag/v3.2.0).
It also is causing new errors in our build chain akin to:
```console
buildDepsnpm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: zysales-web@0.0.4
npm ERR! Found: ember-cli@4.3.0
npm ERR! node_modules/ember-cli
npm ERR!   dev ember-cli@"^4.3.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer ember-cli@"~3.2.0" from ember-cli-dependency-checker@3.3.0
npm ERR! node_modules/ember-cli-dependency-checker
npm ERR!   dev ember-cli-dependency-checker@"^3.2.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/circleci/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2022-04-08T20_28_14_327Z-debug-0.log
```

Any help is appreciated.

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "sinon": "^13.0.0"
   },
   "peerDependencies": {
-    "ember-cli": "~3.2.0 || ^4.0.0"
+    "ember-cli": "^4.0.0"
   },
   "engines": {
     "node": ">= 6"

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "sinon": "^13.0.0"
   },
   "peerDependencies": {
-    "ember-cli": "^4.0.0"
+    "ember-cli": "^3.2.0 || ^4.0.0"
   },
   "engines": {
     "node": ">= 6"

